### PR TITLE
Add label variant to __next__typography

### PR DESCRIPTION
**Pull request by shadoath**
This is probably an oversimplified PR as it stands now. Look to see what needs to be done next.

[Based on Label in this Figma](https://www.figma.com/file/w9lodWeivyS4GHdBNIYaJr/foundations?node-id=2268%3A228)

### CODE REVIEW

**Comment by WilliamKelley** [comment]
It's that simple! However, The variant is missing its specified font-features-settings of cv05, ss03. You'll have to add another argument to `buildVariant` for that.

File: libs/spark/src/theme/__next__typography.ts [hunk 1]
```diff
@@ -39,15 +39,32 @@ const customVariants: Record<__next__TypographyVariant, TypographyStyle> = {
...
-  body: buildVariant(400, 16, 24, undefined, undefined, defaultFontFamily),
-  label: buildVariant(600, 16, 20, undefined, undefined, defaultFontFamily),
+  body: buildVariant(
+    400,
+    16,
+    24,
+    undefined,
+    undefined,
+    defaultFontFamily,
+    "'cv05' on 'ss03 on"
```
Comment (WilliamKelley) [inline libs/spark/src/theme/__next__typography.ts:49]:
```suggestion
    "'cv05' 1, 'ss03' 1"
```

File: libs/spark/src/theme/__next__typography.ts [hunk 2]
```diff
@@ -39,15 +39,32 @@ const customVariants: Record<__next__TypographyVariant, TypographyStyle> = {
...
+  ),
   description: buildVariant(
     400,
     14,
     20,
     undefined,
     undefined,
-    defaultFontFamily
+    defaultFontFamily,
+    "'cv05' on 'ss03 on"
```
Comment (WilliamKelley) [inline libs/spark/src/theme/__next__typography.ts:67]:
```suggestion
    "'cv05' 1, 'ss03' 1"
```

File: libs/spark/src/theme/__next__typography.ts [hunk 3]
```diff
@@ -39,15 +39,32 @@ const customVariants: Record<__next__TypographyVariant, TypographyStyle> = {
...
+  ),
+  label: buildVariant(
+    600,
+    16,
+    20,
+    undefined,
+    undefined,
+    defaultFontFamily,
+    "'cv05' on 'ss03 on"
```
Comment (WilliamKelley) [inline libs/spark/src/theme/__next__typography.ts:58]:
```suggestion
    "'cv05' 1, 'ss03' 1"
```

**Review by WilliamKelley — CHANGES_REQUESTED** [review]
I'm not sure what the syntax rules are for `font-feature-settings`. It seems pretty lax, but let's go with the format from inspecting Inter's [official site](https://rsms.me/inter/#features/ss03): 
image

**Comment by WilliamKelley** [comment]
Also, the font file for Inter bold (600 weight) needs to be brought in, and the font face declaration added (to `__next__fontFaces`). You can see on the Storybook that the letters look way to thin compared to the Figma reference, because it's picking the closest available weight file (400).

Also, the `cv05` feature setting does not seem to be working. I can see `ss03` though. Not sure how to debug that. Maybe something went wrong when I put the files through the web-optimizer on fontsquirrel.com .

**Comment by WilliamKelley** [comment]
More on the [fontsquirrel](https://www.fontsquirrel.com/tools/webfont-generator) generation. I think this is the source of the problem. I've stored the original fonts zips in the repo at `/assets/fonts/zips/Inter...`. Try unzipping that, reuploading to FontSquirrel's Webfont Generator (do like two-at-a-time, it can be finnicky and have a total size limit, so you might have to stop at like 6 files). Choose the "Expert" settings and leave everything as-is, except add what's in the screenshot below:

image

**Comment by shadoath** [comment]
I tried to use FontSquirrel this morning but couldn't get it to take any uploaded files. Is there another way or something you can do for this step?

Also where do I put the Release notes?

**Comment by WilliamKelley** [comment]
Did you try uploading one at a time? If so, I try it myself.

And, `libs/spark/CHANGELOG.md`.

**Comment by shadoath** [comment]
@WilliamKelley Next step is to use the other subset tools you mentioned in the issue?

**Comment by WilliamKelley** [comment]
You definitely can. But I won't hold this PR back if you don't want to. It will be pretty involved and can be considered a separate issue for now. Just revert the initial font file replacement and we'll leave that outstanding.

File: libs/spark/CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,25 @@
...
 ## [vNext](https://github.com/prenda-school/prenda-spark/compare/v1.0.0-alpha.1...vNext) (yyyy-mm-dd)
 
+### **next**
+
+- **\_\_next\_\_typography**
+  - Implemented according to PDS v2.
+  - Preview: `theme.__next__typography.*`
+  - Descriptive changes from current `typography` (PDS v1):
+    - "label"
+      - Added.
```
Comment (WilliamKelley) [inline libs/spark/CHANGELOG.md:12]:
```suggestion
  - Added `font-feature-settings` for "body" and "description" variants.
  - Added "label" variant (with `font-feature-settings`).
```

**Review by WilliamKelley — CHANGES_REQUESTED** [review]
Actually this is fine since there's only the bold(italic) fonts added!

**Review by WilliamKelley — APPROVED** [review]

**Review by slingner — APPROVED** [review]

**Comment by WilliamKelley** [comment]
Reverted because PR title (and resulting squash merge commit) was not in conventional commit style

**Pull request merged**